BrushedMotorControllerFirmware/Kconfigs/Kconfig.position_cntrl

```diff
@@ -1,20 +1,30 @@
 menu "Position Control Configuration"
 
+config POS_CONTROL_ENABLE
+	bool "Enable functions responsible for position control, like mode position, \
+	     pos set, etc.. Continuus calculation of position and pos read is always \
+	     enabled."
+	default y
+
 config KP_NUMERATOR_FOR_SPEED_IN_POS
 	int "proportional part numerator for setting speed during position control"
 	default 4
+	# depends on POS_CONTROL_ENABLE
```

Comment:
why is that commented out?

Reply:
leftover from debugging; corrected, including all implications!